src/test/java/org/apache/sling/jcr/repoinit/PrincipalBasedAclTest.java

```diff
@@ -565,16 +577,16 @@ public void  principalAclNotAvailableRepoLevelPermissions() throws Exception {
             U.parseAndExecute(setup);
 
             // setting up principal-acl will not succeed (principal not located below supported path)
-            // but there exists an equivalent entry with the same definition -> no exception
             setup = "set principal ACL for otherSystemPrincipal \n"
                     + "allow jcr:namespaceManagement on :repository\n"
                     + "end";
-            U.parseAndExecute(setup);
-
-            Principal principal = adminSession.getUserManager().getAuthorizable("otherSystemPrincipal").getPrincipal();
-            for (AccessControlPolicy policy : acMgr.getPolicies(principal)) {
-                assertFalse(policy instanceof PrincipalAccessControlList);
+            try {
+                U.parseAndExecute(setup);
+                fail("Setting a principal ACL outside a supported path must not succeed");
+            } catch (RuntimeException e) {
+                // expected
```

Comment:
see above

Reply:
done in https://github.com/apache/sling-org-apache-sling-jcr-repoinit/pull/14/commits/cfadffb925ae3da29838a5f7a6a15630a147fbec